### DISCUSSION TITLE
Is there a function to autogenerate a control description sheet table after course design?

### DISCUSSION THREAD

**Discussion by acche** [OpenOrienteering/mapper#1882] [Questions & Answers]
Is there a function to auto gem a control description sheet table after course design.

**Comment by dg0yt** [comment]
No, there is no such function. You can draw a control description sheet manually using the course design symbol set. This is meant to cover simple training events, for example. For more advanced course planning features, you have to use a dedicated program like PurplePen, OCAD, or Condes.

The usage of the Course Design Symbol Set is described at https://www.openorienteering.org/mapper-manual/pages/course_design.html.
The development of Course Setting feature is tracked at https://github.com/OpenOrienteering/mapper/issues/242.

**Reply by acche** [reply]
Is there any media tutorial for this usage?
